Automattic/wp-calypso
# Minor tweaks and improvements for the new  Happy Blocks app

#### Proposed Changes

* This PR addresses the last few comments from https://github.com/Automattic/wp-calypso/pull/69324


#### Testing Instructions

1.  Ensure all the comments below [this one](https://github.com/Automattic/wp-calypso/pull/69324#pullrequestreview-1170501621) are fixed.





#### Pre-merge Checklist






Related to #
